grammar.js

```diff
@@ -905,6 +925,99 @@ module.exports = grammar({
       ')',
     ),
 
+    window_partition_by_expression: $ => seq(
```

Comment:
I think `partition_by` is fine here, especially if we can reuse `order_by` as mentioned below